modal/_container_io_manager.py

```diff
@@ -560,6 +560,12 @@ async def memory_restore(self) -> None:
         with restored_path.open("r") as file:
             restored_state = json.load(file)
 
+        # Start a debugger if the worker tells us to
+        if int(restored_state["snapshot_debug"]):
+            logger.debug("Entering snapshot debugger")
+            import pdb
+            pdb.set_trace()
+
```

Comment:
Can you handle cases where `snapshot_debug` is not present? This is necessary because workers take ~24h to rollover so this container entrypoint update will land on a worker that doesn't yet inject that value.